### ISSUE TITLE
Soundswitch and FxSound (DFX Audio Enhancer) issues

### ISSUE THREAD

**Issue by nvypwr** [Belphemur/SoundSwitch#437]
Hi, I have recently updated Soundswitch 4.16 to 5.0.3 but I have encountered an annoying issue. Soundswitch no longer lets me switch between audio devices when I use shortcut keys. It only makes a random switch when I randomly press the combination for 4th or 5th time consecutively. This issue has been prevalent in the recent versions since I hadn't updated Soundswitch for a long time. 4.16 does not have this issue. 

The root cause is a third party software I use called FxSound (Formerly known as DFX Audio Enhancer). This software improves sound quality by installing itself as a driver, then hijacks the default output whenever a change was made. However in one of the recent versions the functionality of Soundswitch seems to be broken.

Basically FxSound integration obstructs keyboard shortcut for switching default output devices for Soundswitch.

This aside I also would like to point out that Soundswitch and FxSound uses same icons in the taskbar while the older version (aka DFX Audio Enhancer) does not. Can you please implement a different icon (Old DFX logo preferred but even FxSound icon with a different color works fine) for Soundswitch to use?  

I'm using Windows 7 Ultimate (x64)

[SoundSwitch.zip](https://github.com/Belphemur/SoundSwitch/files/4603383/SoundSwitch.zip)

**Comment by Belphemur** [comment]
Hello,

As you pointed out, this all FxSound doing.

Every time SoundSwitch change the default device, FxSound come just after and override that change by making its own audio device the default device.

They aren't compatible with each other since they both change the default device.

The old version of SoundSwitch "works" because it remembers what is the last device it switched manually too, but not the one that is actually set as default.

Moreover, about the icon, they are different: 

FxSound, SoundSwitch
![image](https://user-images.githubusercontent.com/197810/81509447-c2fa5b80-92d8-11ea-8d82-14034c585bca.png)

I'm sorry, I'm not going to do anything about this for the moment.

**Comment by nvypwr** [comment]
Oh well, what a shame, guess I'll stick with 4.16 then... I'm still hoping to see an update addressed to this in the future though.

As for icons I meant for "Playback" option. I am aware that Soundswitch icon can be set to "Never".

**Comment by Belphemur** [comment]
For that icon, you have control on it :)

You can go into Playback Devices > Right click on the device you want > Property > Change Icon.

![image](https://user-images.githubusercontent.com/197810/81626587-26b08180-93ca-11ea-9439-26363f471891.png)

SoundSwitch use the icon set in Windows.
